src/main/java/Duke.java

```diff
@@ -1,10 +1,80 @@
+import java.util.Scanner;
+
+
 public class Duke {
+
+    public static void greet() {
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+    }
+
+    public static void lineSeparator() {
+        System.out.println("_________________________");
+    }
```

Comment:
Name of methods should be verb followed by action. For example, printLineSeparator() instead of lineSeparator()